src/MSBuild/XMake.cs

```diff
@@ -3195,6 +3198,33 @@ internal static ISet<string> ProcessWarnNotAsErrorSwitch(CommandLineSwitches com
             return ProcessWarningRelatedSwitch(commandLineSwitches, CommandLineSwitches.ParameterizedSwitch.WarningsNotAsErrors);
         }
 
+        internal static QuestionMode ProcessQuestionModeSwitch(string[] parameters)
```

Comment:
Do we need to force users to choose the mode? Can `/question` default to previous behavior (both - targets and tasks inspected)?

Reply:
Forking this part as a separate PR.

The previous behavior will include both.